### ISSUE TITLE
Interviews with Open Source Designers

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/organization#28]
This was originally thought of in #17 and #21, and I would like to get this going! 

Basically, we can create a list of questions to ask of people active in this community or other open source projects who do a design aspect-y job (content, documentation, onboarding, ui design), and post the interviews on the Medium page and on the Articles in the blog. 

It'd be a great way to spotlight some people working on this and some awesome projects.

Do you want to be featured? Do you know anyone who should be featured?

Question ideas:
* What is it that you do?
* What projects do you contribute to?
* How often do you contribute to open source?
* How did you get started working on open source?
* What's one of the hardest things about working on open source?
* What's one of your favorite things about working on open source?
* What's your favorite typeface? (obvs)
* What's the most recent design book you've read?
* What websites do you go to for inspiration?
* Do you have any suggestions for people interested in joining open source projects? 
* How would they go about joining your project?

Thoughts? Additional questions? Suggestions?

**Comment by elioqoshi** [comment]
I hope it doesn't sound too rude if I'd say that I'd be up for an interview. Would love to talk about Design at Mozilla and my open source design startup :)

**Comment by gillisig** [comment]
@simonv3 sounds like a good idea! I would add a question about their process, like how they handle the community and all that. 

@elioqoshi, I'd love to hear more about your open source design startup, regardless of the interview stuff! :D

**Comment by evalica** [comment]
I like the idea :) So how do you imagine the process? People responding this issue with the answers or creating a PR for the article? Do you want it to be a video or just text? In order to be an interview at least 2 people would need to get involved, or maybe not :)

**Comment by simonv3** [comment]
@evalica I think we could e-mail some people the interview questions and allow them to respond. We could also set up a couple of skype calls if people would rather do that - but I think there's less pressure over e-mail. Of course, I hadn't really considered that people could just respond on here, I guess their answers would be public anyway.

**Comment by andreasn** [comment]
I would like to see an interview with Allan Day from GNOME.
I would also like to see one with Jan from ownCloud.

**Comment by elioqoshi** [comment]
Definitely would love also to see an interview from @bnvk re: Mailpile / QubeOS

**Comment by bnvk** [comment]
> I like the idea :) So how do you imagine the process? People responding this issue with the answers or creating a PR for the article? Do you want it to be a video or just text? 

Awesome idea :+1: @simonv3  thanks for opening it!

I propose letting the person being interviewed choose *how* they would like to be interviewed- some might prefer text + pull request, others into making a self filmed video, others an audio format done in "conversation" style. 

We could even encourage the interviewee to "create" the interview themselves and be creative with it and make "templates" of things like video title cards, etc... to be used for other interviews.

Those are great questions that @simonv3 came up with. Here's a couple more:

- Why do you work on open source projects?
- What are the biggest differences between open and non-open projects?
- Do you see open source as larger cultural movement than just software?
- Are you politically motivated to work on open source projects?

The questions could also be a sort of "pick which you want to answer" model instead of "answer all of these"

@elioqoshi aw, thanks! I'd be happy to :smile:

**Comment by simonv3** [comment]
I'd love to have @una and @GarthDB as an interviewees as well for their work with Design Open.

**Comment by gillisig** [comment]
I'd love to have @simonv3 as an interviewee! ;) 

Again I think this a great initiative and a good way to get more content for Open Source Design. Especially since it's easy for anybody to do in their own time and will probably bring some new visitors to us.

**Comment by simonv3** [comment]
Heh, thanks! 

Also want to nominate @neynah who does all of the awesome design on Sandstorm.

**Comment by neynah** [comment]
@simonv3 It'd be my pleasure!

**Comment by RichardLitt** [comment]
> I'd love to have @simonv3 as an interviewee! ;)

This. 

Great work. Let me know how I can help.

**Comment by RichardLitt** [comment]
Also, it might be worth actually recording these if done over a micraphone, and starting a YouTube series (minimal effort) or a Podcast. Happy to help with this, would be a great project.

**Comment by simonv3** [comment]
Yeah, I think if we're recording them we should definitely also podcast them, but that does require some editing etc. But I've also already heard from a couple of people that they would prefer an e-mail interview - we send them the questions and they reply in their own time. So it'll have to be a mixture of the two.

**Comment by simonv3** [comment]
Alright, first PR for for an interview here: https://github.com/opensourcedesign/opensourcedesign.github.io/issues/57

**Comment by Victorybrown** [comment]
also the interview can be made into an article
